website/project/views/node.py

```diff
@@ -480,17 +513,28 @@ def view_project(auth, node, **kwargs):
 
     if 'forward' in ret['addons']:
         addons_widget_data['forward'] = serialize_forward_widget(node)
-
-    if 'zotero' in ret['addons']:
-        node_addon = node.get_addon('zotero')
-        zotero_widget_data = ZoteroCitationsProvider().widget(node_addon)
-        addons_widget_data['zotero'] = zotero_widget_data
-
-    if 'mendeley' in ret['addons']:
-        node_addon = node.get_addon('mendeley')
-        mendeley_widget_data = MendeleyCitationsProvider().widget(node_addon)
-        addons_widget_data['mendeley'] = mendeley_widget_data
-
+    if not waffle.flag_is_active(request, features.ENABLE_GV):
```

Comment:
nit:
it's better for readability to flip this if statement

Reply:
Done